types/chrome/index.d.ts

```diff
@@ -5242,6 +5242,9 @@ declare namespace chrome.runtime {
         version: string;
     }
 
+    /** Result of the update check. */
+    export type RequestUpdateCheckStatus = 'throttled' | 'no_update' | 'update_available';
```

Comment:
```suggestion
    export interface RequestUpdateCheckStatus = 'throttled' | 'no_update' | 'update_available';
```

Reply:
The suggested change results in syntax error. This is not an interface. A variable with this new type has to be a `string`, not an `object`.

Also, there are 4 more instances of `type` in this file.

Reply:
Makes sense. Thanks for the clarification.